packages/node_modules/@webex/react-container-message-composer/src/container.js

```diff
@@ -177,8 +180,21 @@ export class MessageComposer extends Component {
       } = props;
 
       const files = constructFiles(e.target.files);
-
-      props.addFiles(conversation, activity, files, sparkInstance);
+      const overThreshold = files.some((file) => file.overThreshold);
```

Comment:
Following up on my comment to not store the check as a property:
```suggestion
      const maxSizeExceeded = files.some((file) => file.size >= FILE_ATTACHMENT_MAX_SIZE);
```